pkg/sfu/pacer/pacer.go

```diff
@@ -0,0 +1,31 @@
+package pacer
+
+import (
+	"time"
+
+	"github.com/pion/rtp"
+	"github.com/pion/webrtc/v3"
+)
+
+type ExtensionData struct {
+	ID      uint8
+	Payload []byte
+}
+
+type Packet struct {
+	Header             *rtp.Header
+	Extensions         []ExtensionData
+	Payload            []byte
+	AbsSendTimeExtID   uint8
+	TransportWideExtID uint8
+	WriteStream        webrtc.TrackLocalWriter
+	Metadata           interface{}
+	OnSent             func(md interface{}, sentHeader *rtp.Header, payloadSize int, sentTime time.Time)
+}
```

Comment:
Packet is enqueued with all the data. Extensions are added as necessary and packet sent to `WriteStream`. `OnSent` is the per packet callback. Wondering if that is going to cause performance issues.